pyfolio/utils.py

```diff
@@ -49,10 +49,14 @@
     MONTHLY: MONTHS_PER_YEAR
 }
 
-DEPRECATION_WARNING = ("Data loaders have been moved to empyrical and will "
-                       "be removed from pyfolio in a future release. Please "
-                       "use e.g. empyrical.utils.get_symbol_rets() instead "
-                       "of pyfolio.utils.get_symbol_rets()")
+DATAREADER_DEPRECATION_WARNING = \
+        ("Yahoo and Google Finance have suffered large API breaks with no "
+         "stable replacement. As a result, any data reading functionality "
+         "in empyrical has been deprecated and will be removed in a future "
+         "version."
+         ""
+         "Please use empyrical in the Quantopian Research environment, or "
+         "supply your own data. See README.md for more details.")
```

Comment:
Should we just import this warning message from empyrical?

Reply:
Perhaps we should wait until https://github.com/quantopian/empyrical/pull/97 is at least merged before we do?

Reply:
Turns out that after dealing with the rest of the feedback, nothing in pyfolio ends up being deprecated at all. So this is kind of a refactor PR?